### DISCUSSION TITLE
Account issue

### DISCUSSION THREAD

**Discussion by ntkhang03** [npm/feedback#873] [Registry] [answered]
Hi npm team, I have an account issue and need help

 I don't know why, my account and packages that I have made public suddenly disappeared without any email notification from npm

 My username account is: **ntkhang**
 My email: **nthanhkhang2003@gmail.com**

 Many thanks for the feedback of the team in the future

**Comment by MylesBorins** [comment, marked as answer]
Please open a ticket with support at https://npmjs.com/support

**Reply by ntkhang03** [reply]
I can't login my account to send support to npm

**Comment by MylesBorins** [comment]
You can open a support ticket without being logged in

**Reply by ntkhang03** [reply]
thank you, i saw it
